src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceFactory.char.cs

```diff
@@ -16,25 +16,15 @@ public abstract class ReadOnlySequenceFactoryChar
 
         internal class StringTestSequenceFactory : ReadOnlySequenceFactory<char>
         {
-            static string s_stringData = InitalizeStringData();
-
-            static string InitalizeStringData()
+            public override ReadOnlySequence<char> CreateOfSize(int size)
             {
                 IEnumerable<int> ascii = Enumerable.Range(' ', (char)0x7f - ' ');
+                IEnumerable<int> items = ascii;
+                for(int i = (size + 20) / ascii.Count(); i > 0; i--)
```

Comment:
Why `size + 20`?

Reply:
20 was before. The idea is to generate ReadOnlyMemory that have string inside and have non zero offset (10) and length that less that string length.